core/lib/protobuf_config/src/timestamp_asserter.rs

```diff
@@ -0,0 +1,19 @@
+use anyhow::Context;
+use zksync_config::configs::chain::TimestampAsserterConfig;
+use zksync_protobuf::{required, ProtoRepr};
+
+impl ProtoRepr for crate::proto::config::timestamp_asserter::TimestampAsserter {
+    type Type = TimestampAsserterConfig;
+    fn read(&self) -> anyhow::Result<Self::Type> {
+        Ok(Self::Type {
+            min_time_till_end_sec: *required(&self.min_time_till_end_sec)
```

Comment:
Don't forget to add this config param to stage / other auto-deployed envs; IIUC, they'll get broken otherwise.